src/Tasks/Common/Resources/Strings.resx

```diff
@@ -258,6 +258,9 @@
   <data name="CannotHaveSelfContainedWithoutRuntimeIdentifier" xml:space="preserve">
     <value>It is not supported to build or publish a self-contained application without specifying a RuntimeIdentifier.  Please either specify a RuntimeIdentifier or set SelfContained to false.</value>
   </data>
+  <data name="CannotHaveRuntimeIdentifierPlatformMismatchPlatformTarget" xml:space="preserve">
+    <value>The RuntimeIdentifier platform and the PlatformTarget must match.</value>
```

Comment:
Saying they must match could be understood as meaning that the values need to be the same, which isn't the case.  Can we think of a better wording?